hack/patch-converter-runner/README.md

```diff
@@ -0,0 +1,46 @@
+# Patch Converter Runner 
+
+## Overview
+
+The `patch-converter-runner` program invokes converter library to generate shoot from runtime resources. The purpose of this application is to allow developers to:
+- quickly reproduce problems with updating existing Gardener clusters
+- debug converter without a need to run the whole KIM system
+
+## Building the Binary
+
+To build the `patch-converter-runner` binary navigate to the `hack` folder, and execute the following command:
+
+```sh
+  go build -o ./bin/patch-converter-runner ./patch-converter-runner/cmd/main.go
+```
+
+## Usage
+
+### Options
+
+- `--runtime-path`: The path to the runtime CR file
+- `--shoot-path`: The path to the existing shoot CR file
+- `--kubeconfig-path`: The path to the KCP kubeconfig file
+- `--outputPath`: The path to the output shoot CR file
+
+### Example
+
+To run the `patch-converter-runner` program, execute the following command:
+```sh
+patch-converter-runner -runtime-path=<runtime CR fie path> -shoot-path=<shoot CR path> -kubeconfig-path=<KCP kubeconfig path> -outputPath=<output shoot CR path>
+```
+
+### Helper script
+
+You can also run `./script/run.sh` script to execute the application. Before running the script you need to setup the following environment variables:
+- `KCP_KUBECONFIG` - the path to the KCP kubeconfig file
+- `GARDENER_KUBECONFIG` - the path to the Gardener kubeconfig file
+- `$GARDENER_NAMESPACE` - the namespace where the shoot resources are stored
+
+To run the script:
+1. build the `patch-converter-runner` application
+2. navigate to the `script` folder
+3. execute the following command:
+```sh
+./script/run.sh <output folder> <runtime ID> <shoot name>
```

Comment:
Remove point 2 or

```suggestion
./run.sh <output folder> <runtime ID> <shoot name>
```

Reply:
It should stay as it is since I modified the description